src/raised-button.jsx

```diff
@@ -15,6 +13,92 @@ function validateLabel(props, propName, componentName) {
   }
 }
 
+function getStyles(props, state) {
+  const {
+    baseTheme,
+    button,
+    raisedButton,
+  } = state.muiTheme;
+
+  const {
+    disabled,
+    icon,
+    labelPosition,
+    primary,
+    secondary,
+  } = props;
+
+  const amount = (primary || secondary) ? 0.4 : 0.08;
+
+  let backgroundColor = raisedButton.color;
+  let labelColor = raisedButton.textColor;
+
+  if (props.disabled) {
+    backgroundColor = props.disabledBackgroundColor || raisedButton.disabledColor;
+    labelColor = props.disabledLabelColor || raisedButton.disabledTextColor;
+  } else if (props.primary) {
+    backgroundColor = raisedButton.primaryColor;
+    labelColor = raisedButton.primaryTextColor;
+  } else if (props.secondary) {
+    backgroundColor = raisedButton.secondaryColor;
+    labelColor = raisedButton.secondaryTextColor;
+  }
```

Comment:
All my nagging have finally paid off :heart_eyes: :laughing: :laughing: 

Although the extra `props.` seems unnecessary O.o